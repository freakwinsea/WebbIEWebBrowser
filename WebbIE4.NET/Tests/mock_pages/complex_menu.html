<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Complex Menu Test</title>
</head>
<body>
    <nav>
        <ul>
            <li><a href="/home">Home</a></li>
            <li>
                <a href="/products">Products</a>
                <ul>
                    <li><a href="/products/desktops">Desktops</a></li>
                    <li>
                        <a href="/products/laptops">Laptops</a>
                        <ul>
                            <li><a href="/products/laptops/gaming">Gaming Laptops</a></li>
                            <li><a href="/products/laptops/business">Business Laptops</a></li>
                        </ul>
                    </li>
                </ul>
            </li>
            <li><a href="/about">About Us</a></li>
            <li><a href="/contact">Contact</a></li>
        </ul>
    </nav>
</body>
</html>
