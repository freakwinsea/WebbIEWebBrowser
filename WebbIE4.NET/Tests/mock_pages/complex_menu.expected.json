[
  {
    "id": "webbie-id-0",
    "type": "link",
    "text": "Home",
    "href": "/home"
  },
  {
    "id": "webbie-id-1",
    "type": "link",
    "text": "Products",
    "href": "/products"
  },
  {
    "id": "webbie-id-2",
    "type": "link",
    "text": "Desktops",
    "href": "/products/desktops"
  },
  {
    "id": "webbie-id-3",
    "type": "link",
    "text": "Laptops",
    "href": "/products/laptops"
  },
  {
    "id": "webbie-id-4",
    "type": "link",
    "text": "Gaming Laptops",
    "href": "/products/laptops/gaming"
  },
  {
    "id": "webbie-id-5",
    "type": "link",
    "text": "Business Laptops",
    "href": "/products/laptops/business"
  },
  {
    "id": "webbie-id-6",
    "type": "link",
    "text": "About Us",
    "href": "/about"
  },
  {
    "id": "webbie-id-7",
    "type": "link",
    "text": "Contact",
    "href": "/contact"
  }
]
